block-streamer/Cargo.toml

```
@@ -13,11 +13,15 @@ borsh = "0.10.2"
 cached = "0.49.3"
 chrono = "0.4.25"
 futures = "0.3.5"
+graphql_client = "0.14.0"
 lazy_static = "1.4.0"
 mockall = "0.11.4"
+near-lake-framework = "0.7.8"
 prometheus = "0.13.3"
 prost = "0.12.3"
 redis = { version = "0.21.5", features = ["tokio-comp", "connection-manager"] }
+registry-types = { path = "../registry/types" }
+reqwest = { version = "^0.12.4", features = ["json", "blocking"] }
```

```suggestion
reqwest = { version = "0.11", features = ["json"] }
```
- `blocking` shouldn't be necessary since we're using `async`?
- `0.11` is the version used internally to `graphql_client` so this makes them compatible